src/index-metadbs/mpds/v1/links.json

```diff
@@ -0,0 +1,34 @@
+{
+   "meta" : {
+       "api_version" : "1.0.0",
+       "query" : {
+           "representation" : "/links"
+       },
+       "more_data_available" : false,
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : [
+      {
+         "id" : "mpds",
+         "type" : "links",
+         "attributes" : {
+            "name": "Materials Platform for Data Science",
+            "description": "A highly curated Pauling File dataset based on ~0.5M publications and backing up Springer Materials, ICDD PDF, ASM APD, MedeA, Pearson Crystal Data, AtomWork Advanced, etc.",
+            "base_url": "https://api.mpds.io",
```

Comment:
I get a 301 permanent redirect when querying this URL; is this correct? I get the same result even if I supply a nonsense `"Key"` in the HTTP headers.